generators/assets/cli/en.yaml

```diff
@@ -255,3 +255,9 @@ cli:
     description: List, create, update or delete VPGs.
   deprecated-api: Deprecated API calling.
   alternative-api-suggestion: "Please consider to use the alternative API (command: `%s`)."
+  self_update:
+    summary: Updates soracom-cli to the latest version.
+    description: Updates soracom-cli to the latest version.
+    already_latest: "It doesn't need to update the CLI because the using version is the latest: %s\n"
+    prompt_confirmation: "There is a newer soracom-cli version '%[1]s'.\nRelease Note: https://github.com/soracom/soracom-cli/releases/tag/%[1]s\nDo you want to continue updating? (y/N) "
```

Comment:
How about adding some sentences to encourage the users to use appropriate package managers to update the binary, like:

```
There is a newer soracom-cli version '$VERSION'.
Release Note: http://github.com/soracom/soracom-cli/releases/tag/$VERSION

If you installed soracom-cli using a package manager, please use it to update.

  brew update soracom-cli
    or
  snap refresh soracom
    or
  dpkg -i soracom_$VERSION_$ARCH.deb

Do you wan to continue updating? (y/N) 
```

Reply:
Awesome, that seems helpful. I added that notes @ [8974235](https://github.com/soracom/soracom-cli/pull/51/commits/8974235d97c6f89245fdc89c9eff8f5a42f4180e)